### DISCUSSION TITLE
blog/What-is-a-GITHUB-Profile-Readme

### DISCUSSION THREAD

**Discussion by giscus** [MaheshBabu11/personal-blog#4] [General]
# blog/What-is-a-GITHUB-Profile-Readme

Read this article to find out how you can create a beautiful readme for your github profile.

https://blog.maheshbabu11.dev/blog/What-is-a-GITHUB-Profile-Readme

**Comment by tarunjain1st** [comment]
Hi Mahesh, do you know how to add images to github readme using path?

**Reply by MaheshBabu11** [reply]
Hi @tarunjain1st , You can add it like this for example 

```
<img src="https://github.com/MaheshBabu11/MaheshBabu11/blob/main/profile-3d-contrib/profile-night-rainbow.svg"/>
```

You can upload the image to ur reop and then use that path.
Hope that resolves your issue 🙂

**Reply by tarunjain1st** [reply]
it's not working for me no image is visible can you try on a readme once

**Reply by MaheshBabu11** [reply]
Can you check this out, i have the same on my profile readme 

https://github.com/MaheshBabu11/MaheshBabu11/blob/main/README.md
